### DISCUSSION TITLE
Where does Grott log to?

### DISCUSSION THREAD

**Discussion by mandelorean** [johanmeijer/grott#535] [General]
Hey, apologies for noob question, but where is grott logging to?

It all looks like its running, but it doesnt appear to be connecting to MQTT. So just wanted to check logs to see if there's any hints in there.

**Comment by johanmeijer** [comment]
How are you running Grott.

As a service the log is written to the system log (can be viewed with the journalctl. Command).

In a docker container the log is written to the container log and can be accessed by the docker logs command.

**Comment by mandelorean** [comment]
Docker - and yes that worked. Thanks
